web/css/tour.css

```diff
@@ -22,6 +22,10 @@
   width: 450px;
 }
 
+.joyride-close-tip {
```

Comment:
We didn't have custom `.joyride` CSS before, and [we're including the Joyride stylesheet in this branch already](https://github.com/nasa-gibs/worldview/blob/module-loaders-tour/web/css/main.css#L10).

We shouldn't have to add custom CSS to our stylesheets to achieve style parity with production. @ZachTRice @Benjaki2 I might have missed something when I included the dependency stylesheets. Maybe the paths aren't correct, or maybe they're in the wrong order or something. Both of you have added dependency styles to stylesheets in your PRs, but I think maybe there's a problem with how I did the CSS originally. Have either of you looked into that?

Reply:
The plugin we were using before was modified to add css directly to the plugin. Here is the css blame: https://github.com/nasa-gibs/worldview/blame/master/web/ext/tour/joyride-2.0.3-3/joyride.css